spotify-app/demiv2/script/main.js

```diff
@@ -0,0 +1,156 @@
+    $(".section-tracks").hide()
+    
+      artists.artists.items.forEach(function(item) {
+        
+        const imagesUrl = item.images.length !=0 ? item.images[0].url : '';
+
+        boxCard+="<div class='col p-2 card'>";
+        boxCard+="<div class='photobox photobox-artists photobox_type16' data-url='https://api.spotify.com/v1/artists/"+item.id+"/albums'>";
+        boxCard+="<div class='photobox__previewbox'>";
+        boxCard+="<img class='img-fluid photobox__preview card-img' src="+imagesUrl+">";
+        boxCard+="<span class='photobox__label'>"+item.name+"</span>";
+        boxCard+="</div>";
+        boxCard+="</div>";
+        boxCard+="</div>";
+        
+      });
+
+    $("section.section-artists .row").html(boxCard);
+    $("section.section-artists .row").show();
+    $(".section-artists").show()
+
+  }
+
+  // llamamos al siguiente pantel donde le pasaremos 
+  // los mismos datos de token time out base url... 
+  // pero las funciones seran ditintas y la query
+
+  /**
+   *  Enviamos datos a la api de sporify
+   */
+  $("body").on("click", ".photobox-artists", (function(e) { 
+    const query =  $(this).attr("data-url")
+    spotifyApi.retrieveAlbums(query, showAlbums, showError)
+  }))
+  
+  //definimos la funcion showAlbums que nos servira para enseñar los albums
+  function showAlbums(albums) {
+
+    $(".section-artists").fadeIn(1000).hide();
+    $(".section-album").fadeIn(1000).show();
+
+    let boxAlbums = "";
+
+    albums.items.forEach(function(item) {
+
+        boxAlbums+="<div class='col p-2 card'>";
+        boxAlbums+="<div data-url='https://api.spotify.com/v1/albums/"+item.id+"/tracks' class='photobox photobox-album photobox_type16'>";
+        boxAlbums+="<div class='photobox__previewbox'>";
+        boxAlbums+="<img class='img-fluid photobox__preview' src="+item.images[0].url+">";
+        boxAlbums+="<span class='photobox__label'>"+item.name+"</span>";
+        boxAlbums+="</div>";
+        boxAlbums+="</div>";
+        boxAlbums+="</div>";
+
+      (item.images.length != 0) ? boxAlbums : "";
+
+    });
+
+    let boxtitles = ""
+    albums.items.forEach(function(item) {
+      boxtitles+="<span class='subsection-titles'>"+item.name+" / "+"</span>";
+    });
+    
+    $("section.section-album .row.gallery").html(boxAlbums);
+    $("section.section-album .row.titles div").html(boxtitles);
+
+  }
+
+  /**
+   *  Enviamos datos a la api de sporify
+   */
+  $("body").on("click", ".photobox-album", (function(e) { 
+    const query =  $(this).attr("data-url")
+    spotifyApi.retrieveTracks(query, showTracks, showError)
+  }))
+
+  //definimos la funcion showTracks que nos servira para enseñar las canciones
+  function showTracks(tracks) {
+
+    $(".section-album").hide();
+    $(".section-tracks").fadeIn(1000).show();
+  
+    let tracktitles = ""
+  
+    tracks.items.forEach(function(item) {
+      tracktitles+="<span class='subsection-tracks' data-url='"+item.id+"'>"+item.name+" "+"</span>";

```

Comment:
yep! you can use `string literals` if you want, is more easy to read

Reply:
yeah! sure, i'll do it, yesterday i didn't understand how to do, but... now with  a youtube tutorial about 20 minilessons about js6 i do!